README.md

```diff
@@ -9,7 +9,7 @@ DUS leverages the power of Amazon Textract, Amazon Comprehend , Amazon Comprehen
 ![img](./images/DUS_Arch.png)
 
 Current document formats supported : **PDF,JPG,PNG**
-
+Current maximum document file size supported : **150MB**
```

Comment:
add a comment in here about maximum number of file uploads

Reply:
updated